tests/classification/interpret/sst_test.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+
+
+def test_gradient_visualization():
+    from allennlp.predictors.predictor import Predictor
+
+    predictor = Predictor.from_path(
+        "https://storage.googleapis.com/allennlp-public-models/sst-roberta-large-2020.06.08.tar.gz"
+    )
+    sentence = "a very well-made, funny and entertaining picture."
+
+    inputs = {"sentence": sentence}
+    from allennlp.interpret.saliency_interpreters import SimpleGradient
+
+    simple_gradient_interpreter = SimpleGradient(predictor)
+    simple_gradient_interpretation = simple_gradient_interpreter.saliency_interpret_from_json(
+        inputs
+    )
+
+    gradients = simple_gradient_interpretation["instance_1"]["grad_input_1"]
+    assert max(gradients) - min(gradients) < 0.75
```

Comment:
Why do we check this difference? And where do we get `0.75` from?

Reply:
When this was broken, the output was all `0` except for one cell that was `1`. I just want to check whether there is some variety in the numbers. Since they all add up to `1`, this seemed like a good check. It's possible that checking for just the `max` would be good enough.